app/models/concerns/has_custom_fields.rb

```diff
@@ -114,7 +114,9 @@ def append_custom_field(target, key, value)
       get_custom_field_descriptor(key).append_field(target, key, value)
     end
 
-    def register_custom_field_type(name, type, max_length: DEFAULT_FIELD_DESCRIPTOR.max_length)
+    def register_custom_field_type(name, type, max_length: nil)
```

Comment:
What happens if I pass in a max_length greater than the DEFAULT_FIELD_DESCRIPTOR?

Reply:
That's allowed.